lib/utils/parseList.js

```diff
@@ -27,7 +29,14 @@ function getParseDetailList (opts) {
         lang: opts.lang,
         country: opts.country,
         cache: opts.cache
-      });
+      })
+        .then((app) => {
```

Comment:
this is coupled with the fullDetail option; note in line 9/11, if fullDetail is false the permissions options will be ignored